client/src/components/RepoPreview.tsx

```diff
@@ -25,11 +26,11 @@ export const RepoPreview: React.FC<{ repo: RepoDTO, orgNames: Map<number, string
                         {/* Badge */}   
                         {repo && repo?.badge !== RepositoryBadge.none &&
                             <span
-                                className={`badge rounded-pill ${RepositoryService.BadgeToBootstrapColor(repo?.badge)}`}
+                                className={`badge rounded-pill ${BadgeUtils.toBootstrapColor(repo?.badge)}`}
                                 style={{ fontSize: '0.7rem', marginLeft: '1em' }}
                             >
-                                <i className={`bi ${RepositoryService.BadgeToHumanBootstrapIcon(repo?.badge)}`}> </i>
-                                {RepositoryService.BadgeToHumanText(repo?.badge)}
+                                <i className={`bi ${BadgeUtils.toBootstrapIcon(repo?.badge)}`}> </i>
+                                {BadgeUtils.toHumanText(repo?.badge)}
```

Comment:
Do this in `RepoOfUser.tsx`, too. When I tried to click on the filter button on that page I got an error because of the missing function.